scripts/train/train.py

```diff
@@ -160,8 +162,28 @@ def main(cfg):
 
     # Build Model
     print('Initializing model...')
-    with init_context:
-        model = build_composer_model(cfg.model, tokenizer)
+    # if cfg.peft exists and is lora
+    if cfg.get('lora', None) is not None:
```

Comment:
I think this block should be in `build_composer_model`

I'm wondering if we can clean this block and the big `if` block in the init of `ComposerHFCausalLM` by having `build_composer_model` call a new `build_lora_model` function under the appropriate conditions?

Reply:
yeah sounds good!